src/components/_proxied-dot-io/packer/remote-plugin-docs/server.ts

```diff
@@ -57,7 +62,19 @@ async function generateStaticProps({
 		remotePluginsFile,
 		currentPath,
 	})
-	const navNode = getNodeFromPath(currentPath, navData, localContentDir)
+	// Attempt to match a navNode for this path.
+	// Note that we may not be able to find a match, in which case we 404.
+	let navNode
+	try {
+		navNode = getNodeFromPath(currentPath, navData, localContentDir)
+	} catch (e) {
+		const isNotFound = String(e).indexOf('Missing resource') !== -1
```

Comment:
```suggestion
		const isNotFound = String(e).includes('Missing resource')
```

⛏️ super minor, but reads a bit clearer